### DISCUSSION TITLE
Container Metadata

### DISCUSSION THREAD

**Discussion by vsoch** [supercontainers/containers-wg#6] [Ideas]
This is a discussion thread for the [Container Metadata](https://supercontainers.github.io/containers-wg/ideas/container-metadata/) idea. You can:

 - vote this discussion up to express interest in the idea
 - add comments, feedback, or ideas for discussion in this thread.

**Comment by sk8forether** [comment]
Opened #14  to add some thoughts to this idea.  Apologies if that's not the right process, just wanted to note them down somewhere.

**Reply by vsoch** [reply]
It's perfect!
